src/pages/community-champions.js

```diff
@@ -370,6 +376,100 @@ const HowToQualifyCard = ({ icon, title, description }) => (
     </HowToQualifyCardContainer>
 );
 
+const Line = styled('hr')`
+    border-color: ${({ theme }) => theme.colorMap.greyDarkOne};
+    margin-bottom: ${LINE_BOTTOM_MARGIN};
+    @media ${screenSize.upToMedium} {
+        margin-bottom: ${size.large};
+    }
+`;
+
+const GreenBulletedListWithGreyText = styled(GreenBulletedList)`
+    grid-area: bullets;
+    margin: ${size.default} 0 0;
+    max-width: ${OTHER_DETAILS_AND_REQUIREMENTS_LIST_MAX_WIDTH};
+    li {
+        margin-bottom: ${size.xsmall};
+    }
+`;
+
+const OtherDetailsAndRequirementsContainer = styled('div')`
+    display: grid;
+    grid-template-areas:
+        'image text'
+        'image bullets';
```

Comment:
Idk about you but this is awesome, grid template areas is my fav CSS feature bc it is so handy! Awesome job!